### DISCUSSION TITLE
Save Pattern Names and Nothing Else

### DISCUSSION THREAD

**Discussion by luciferseamus** [bangcorrupt/hacktribe#340] [Q&A]
Hello everyone. I was wondering if there is a quick way (one of the scripts included with hacktribe-editor perhaps?) to export/modify patterns which would allow them to retain the name (and location info) but strips away everything else.

The reason I ask:
I was running into the polyphony wall trying to use too many voices on my original grey E2. So, I picked up a second one (blue) to use as accompaniment. What I would like to do is export all patterns from my original E2, and import them into my 2nd E2 but have all triggers, motions, fx, and knob orig values set to some default value.  
I know how to do this the slow way but it would be fantastic to not have to sit with the E2 going through each and every pattern to  clear the information by hand.  I can do it but I keep thinking there must be an easier way.

Bonus points:
-Interactive way to manipulate the patterns
-Selecting which parts I want to keep and which to clear.
-A way to name them off of an E2

I do realize that is asking a lot but I figured I'd try.

Thanks in advace!
-Aaron

**Comment by bangcorrupt** [comment]
I don't have any pattern editing scripts, it's on an ever growing todo list.  The pattern name is 16 characters long and starts at byte 0x110.
